prime-router/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml

```diff
@@ -35,6 +55,16 @@ elements:
     value: [ '""' ]
     hl7Spec: [ '/PATIENT_RESULT/ORDER_OBSERVATION/OBR-16-1' ]
 
+  - name: ak-specimen-source-site-text
+    condition: true
+    value: [ 'Bundle.entry.resource.ofType(Specimen).collection.bodySite.text' ]
+    hl7Spec: ['/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM-8-2']
+
+  - name: ak-specimen-source-site-original-text
+    condition: true
+    value: [ '""' ]
+    hl7Spec: ['/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM-8-9']
+
```

Comment:
Did AK  asked to set these SPM-8-9,PID-22-9,PID-10-9 to blank? I don't see that in the email